docs/building.md

```diff
@@ -110,6 +110,31 @@ cmake --build build -j $(sysctl -n hw.ncpuonline)
 
 <details><summary>Windows via MinGW</summary>
 
+<details><summary>Installing Windows Subsystem for Linux</summary>
+
+If you are building on Windows and do not have WSL already setup this will install WSL and Ubuntu by default (Requires Windows 10 2004 or higher or Windows 11)
```

Comment:
Thanks, that was helpful :)

![image](https://user-images.githubusercontent.com/204594/165213721-a4dd46ef-dc8f-446b-a12e-af9a6d46c9f7.png)